src/Api/Security/AdminAuthenticationProvider.php

```diff
@@ -200,6 +199,16 @@ public function getAdminToken(): string
         return $this->cacheProvider->fetch($cacheKey);
     }
 
+    public function getMboJWT(): string
```

Comment:
Maybe we should cache the token as it doesn't change until the shop is re-register. And clean cache when on of its parameters change